src/overview.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: page
+title: Overview
+---
+
+# Overview
+
+Workbox is a collection of libraries and build tools that make it easy to
+save your website's resources locally, on your users' devices. Consider Workbox
+if you want to:
+
+* Make your site work offline.
+* Improve load performance. You can use Workbox to save and serve common
+  resources locally, rather than from the network.
+
+The only difference between the two options is whether you save all or only
+some resources locally.
+
+## The technology behind Workbox
+
+Modern offline web apps are possible thanks to **service workers**. Service
+workers are background workers, written in JavaScript, that can locally save
+all of the resources needed to run your site offline.
+
+You can think of a service worker as a [proxy server][proxy] that's running in
+the background of your users' devices. Your site makes a request for a resource,
+the service worker intercepts the request, and then decides whether to serve a
+local version of the resource or to fetch a fresh version from the network.
+
+[proxy]: https://en.wikipedia.org/wiki/Proxy_server
+
+The place where Workbox saves resources locally is called the **cache**.
```

Comment:
Would you be open to changing this to `locally is called the (Service Worker) **cache**`? I think the simplicity of just saying cache and caching makes it a lot more beginner friendly, but worry that we might be setting them up for confusion between the HTTP cache (where all web content is cached) and the Service Worker cache / Cache Storage (where only items from Workbox/Service Worker scripts get cached).